qdrant-landing/content/articles/io_uring.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Qdrant under the hood: io_uring"
+short_description: "The linux io_uring API offers great performance in certain cases. Here's how Qdrant uses it!"
```

Comment:
I've some capital letter nitpicks. I believe it is good to be consistent.

I'm sure you'll be doing another pass to fix these before publishing, but I still hope they're useful :smile: 

```suggestion
short_description: "The Linux io_uring API offers great performance in certain cases. Here's how Qdrant uses it!"
```